packages/docs/src/components/content/StorybookExample.tsx

```diff
@@ -62,7 +62,7 @@ const StorybookExample = ({ theme, componentName, minHeight, storyId }: Storyboo
   // when the iframe's content loads, calculate height of iframe & set html
   const onIframeLoad = () => {
     if (iframeRef.current) {
-      setIframeHeight();
```

Comment:
Removing the parens made it so iframes could load with a non-zero height. Don't know why!

Reply:
It's because calling the function was setting the height to 0. When we don't call it, though, the height is wrong, just not zero. I had to add some logic to wait until the content had truly loaded (the story rendered by the storybook client) before setting the height.